packages/fxa-payments-server/server/lib/routes/post-metrics.test.js

```diff
@@ -31,27 +31,37 @@ const invalidBody = {
 };
 
 describe('post-metrics route', () => {
-  test('POST valid input should return 200', done => {
-    request(app)
-      .post('/metrics')
-      .send(validBody)
-      .set('Accept', 'application/json')
-      .expect(200)
-      .end((err, res) => {
-        if (err) return done(err);
-        done();
-      });
+  // eslint-disable-next-line jest/expect-expect
+  test('POST valid input should return 200', () => {
+    return new Promise(done => {
+      request(app)
+        .post('/metrics')
+        .send(validBody)
+        .set('Accept', 'application/json')
+        .expect(200)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
```

Comment:
I don't have a lot of experience with async Jest tests, but based on reading [the docs](https://jestjs.io/docs/en/asynchronous), it seems that resolving the promise with an error string will not fail the test. This should be `reject(err)`, where line 36 would change to `return new Promise((resolve, reject) => {`.